src/controller.rs

```diff
@@ -403,14 +403,57 @@ fn pod_template(config: &WasmCloudHostConfig, _ctx: Arc<Context>) -> PodTemplate
         ..Default::default()
     }];
 
-    let mut volume_mounts = vec![VolumeMount {
+    let mut nats_volume_mounts = vec![VolumeMount {
         name: "nats-config".to_string(),
         mount_path: "/nats/nats.conf".to_string(),
         read_only: Some(true),
         sub_path: Some("nats.conf".to_string()),
         ..Default::default()
     }];
 
+    let mut wasm_volume_mounts: Vec<VolumeMount> = vec![];
+    if let Some(certificates) = &config.spec.certificates {
+        if let Some(authorities) = &certificates.authorities {
```

Comment:
You can remove one level of `if` nesting here by using [`.and_then(...)`](https://doc.rust-lang.org/std/option/enum.Option.html#method.and_then), like so:
```rust
    if let Some(authorities) = &config.spec.certificates.and_then(|certificates| certificates.authorities) {
```

Here's a full worked example: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f490be957f531a48f12f70d171e16586